### ISSUE TITLE
A11y_mssql for VSCode_Results Tab_Keyboard: Grid present under "Results" section is not accessible using keyboard

### ISSUE THREAD

**Issue by InduPriya1805** [microsoft/vscode-mssql#1230]
GitHubTags: #A11y_mssql for VSCode; #A11yMAS; #A11yTCS; #C+E-H; #DesktopApp; #Win10; #Keyboard; #MAS2.1.1; #mssql for VSCode

**Environment Details:**
Visual Studio Code – Insider (install from https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fcode.visualstudio.com%2Finsiders.&data=02%7C01%7C%7C00d9f7de52714b323f9908d607afe0c5%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636704849297354687&sdata=cmMqckjoNPGtCS3SZ00bCqCnFprKgiux%2BIjyvzgU0hE%3D&reserved=0)
Version: 1.34.0-insider (user Setup)
Mssql version: 1.6.0
OS Version: Microsoft Windows 10 Enterprise (10.0.17763 Build 17763)

**MAS ID Violated:** MAS 2.1.1

**Repro Steps:**
1. Open Visual Studio Code in your system
2. Navigate to "Extensions" and install "SQl Server (mssql)" extension
3. Press F1 and select MS SqL : Connect and connect to any sql server
4. "Write  code (SELECT *
FROM sys.objects;) in the new document and press F1 and select ""MS SQL : Execute Query""  "
5. Verify Grid present under "Results" is accessible using keyboard or not

**Actual Result:**
Grid present under "Results" section is not accessible using keyboard

**Expected Result:**
Grid present under "Results" section should be accessible using keyboard

**User Impact:**
Keyboard user are unable to see/view the data present in the grid if it is not accessible using keyboard

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

**Attachments:**
[Bug_1230_Grid is not accessible using keyboard.pptx](https://github.com/Microsoft/vscode-mssql/files/3111152/Bug_1230_Grid.is.not.accessible.using.keyboard.pptx)

**Comment by kenvanhyning** [comment]
Cleaning up fixed accessibility issues.

**Comment by BhanuPrakashGit** [comment]
Github:#A11ySev1;

**Comment by fsteffi** [comment]
#Closed;
Version: 1.79.0-insider (user setup)
Commit: fa8eefda18bf099d82b58453f8fe5d3010196fa0
Date: 2023-05-31T03:06:45.096Z
Electron: 22.5.4
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Windows_NT x64 10.0.25370
